main.py

```diff
@@ -1,7 +1,32 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
```

Comment:
Add a blank line here to logically isolate different commands and improve code perception

Reply:
blank after genres = ["Western", "Action", "Dramma"] or after Genre.objects.create(name=genre)?